### ISSUE TITLE
Specify default environment variable values in codegen.yml

### ISSUE THREAD

**Issue by steebchen** [dotansimha/graphql-code-generator#1812]
It would be awesome if you could specify a default value for environment variables, like you can usually do with yaml files such as in docker:

```yaml
version: '3'

services:
  lb:
    image: nginx:latest
    environment:
      PASSWORD: "${PASS:-secret}"
```

**Comment by dotansimha** [comment]
Implemented in: https://github.com/dotansimha/graphql-code-generator/pull/2016

**Comment by steebchen** [comment]
Thanks!

**Comment by dotansimha** [comment]
Available in 1.3.0 🎉
